pkgs/top-level/all-packages.nix

```
@@ -28175,6 +28175,8 @@ with pkgs;
 
   rofi-pulse-select = callPackage ../applications/audio/rofi-pulse-select { };
 
+  rofi-rbw = callPackage ../applications/misc/rofi-rbw { };
```

```suggestion
  rofi-rbw = python3Packages.callPackage ../applications/misc/rofi-rbw { };
```

From @gador recommendations, this would be also needed

---

thanks :+1: